orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1003,6 +1003,39 @@ def run_step(model, input):
 
         _test_helpers.assert_values_are_close(ort_prediction, pt_prediction)
 
+@pytest.mark.parametrize("operator", ['min', 'max'])
+@pytest.mark.parametrize("dim", [None, 0, -1])
+@pytest.mark.parametrize("keepdim", [True, False])
+def test_gradient_correctness_max(operator, dim, keepdim):
+    func = torch.min if operator == 'min' else torch.max
+
+    class NeuralNetMax(torch.nn.Module):
+        def forward(self, input):
+            if dim is None:
+                return func(input)
```

Comment:
I am thinking adding one nore func to consume the outputs inside toe module is better. As max and max.dim are returning different number of outputs.